timeout_initialization: 10
timeout_listen_firt_ping: 10
timeout_go_flare: 60
timeout_bump_flare: 10
timeout_finalize: 10
timeout_regul: 3
start_depth: 1.0
heading_delta: 30
thust_far: 0.2
thrust_middle: 0.1
thrust_finalize: 0.1
far_border: 10.0
infinity: 1.0e5
middle_border: 4.0
close_border: 0.2
pings_needed: 3
close_koef: 0.15
max_close_thrust: 0.05